locale/en.json

```diff
@@ -12,6 +12,8 @@
   "LABEL": "Label",
   "TYPE": "Type",
   "FOREST": "Forest",
+  "STAGE": "Stage",
+  "NO_PENDING_PUBLIC_SIGHTINGS": "There are no pending public sightings",
```

Comment:
Does this need to be changed too?

Reply:
i left it for now so that it was clearer why the table was empty. we can adjust when contributor sightings are there as well to say something like "there are no pending public or contributor sightings"